src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersControllerIT.java

```diff
@@ -874,6 +879,7 @@ private void verifyDbCpoAuditValues(UpdateCasePaymentOrderRequest request,
             assertEquals(CREATED_BY_IDAM_MOCK, latestRevision.getEntity().getCreatedBy());
             assertNotNull(latestRevision.getEntity().getCreatedTimestamp());
             assertTrue(previousCreatedTimestamp.isBefore(latestRevision.getEntity().getCreatedTimestamp()));
+            assertTrue(latestRevision.getEntity().isHistoryExists());
```

Comment:
Change to `assertEquals(HISTORY_EXISTS_UPDATED, updatedEntity.get().isHistoryExists());`  rather than assertTrue: to highlight that we expect it to be changed.